frontend/src/components/RoleMap/RoleMapForm.module.css

```diff
@@ -0,0 +1,16 @@
+.editButtonContainer {
+    display: flex;
+    justify-content: center;
+    margin-top: 16px;
+    gap: 8px;
+}
+
+.addPermissionButton {
+  margin-top: 8px;
+  margin-left: 4px;
+    border-color: #1677ff;
```

Comment:
```suggestion
```

Reply:
I don't like that this button is constantly glowing blue, after this change it should be only blue on hover